CMake/SlicerCPack.cmake

```diff
@@ -210,7 +210,11 @@ set(CPACK_NSIS_MODIFY_PATH OFF)
 
 set(APPLICATION_NAME "${Slicer_MAIN_PROJECT_APPLICATION_NAME}")
 set(EXECUTABLE_NAME "${Slicer_MAIN_PROJECT_APPLICATION_NAME}")
-set(CPACK_PACKAGE_EXECUTABLES "..\\\\${EXECUTABLE_NAME}" "${APPLICATION_NAME}")
+set(PACKAGE_APPLICATION_NAME "${APPLICATION_NAME}")
+if(Slicer_VERSION_TWEAK GREATER 0)
```

Comment:
Consider the definition of release vs non-release defined [here](https://github.com/Slicer/Slicer/blob/6297a07267540e972d42cddfdefa7f2b7dc8d1e7/CMakeLists.txt#L473-L475), it seems we would like the full version to be associated with the application name in that situation:
```
if(NOT DEFINED Slicer_VERSION_TWEAK)
```